packages/preferences/src/browser/preference-frontend-module.ts

```diff
@@ -32,14 +33,18 @@ export const PreferencesWidgetFactory = Symbol('PreferencesWidgetFactory');
 export function bindPreferences(bind: interfaces.Bind, unbind: interfaces.Unbind): void {
     bindPreferenceProviders(bind, unbind);
 
-    bindViewContribution(bind, PreferencesContribution);
-
     bind(PreferencesContainer).toSelf();
```

Comment:
Do we still need `PreferenceContainer`? Old widgets should be removed to clean up the codebase from dead code.

Reply:
I have removed everything that doesn't seem necessary from the frontend module and I have removed the files that don't seem to be in use anymore after that.